_data/sidebars/marketplace_dev_sidebar.yml

```diff
@@ -51,6 +51,12 @@ entries:
           - title: Marketplace Order Management
             url: /docs/marketplace/dev/feature-walkthroughs/marketplace-order-management.html
 
+          - title: Marketplace Promotions and Discounts
+            url: /docs/marketplace/dev/feature-walkthroughs/marketplace-promotions-and-discounts.html
```

Comment:
the link should be /docs/marketplace/dev/feature-walkthroughs/marketplace-promotions-and-discounts-feature-walkthrough.html. Please adjust the link in editor accordingly